src/bundle.rs

```diff
@@ -287,15 +318,58 @@ impl<T: Authorization> Bundle<T> {
 
 /// Authorizing data for a bundle of actions, ready to be committed to the ledger.
 #[derive(Debug)]
-pub struct Authorized {
-    proof: Proof,
-    binding_signature: redpallas::Signature<Binding>,
+pub struct BundleAuth {
```

Comment:
What is the purpose of this structure as a standalone entity?

Reply:
Hmm, from the perspective of a transaction parser, I think you can leverage the new `context: &mut R` fields on `Bundle::try_authorize` parameters to parse the spendAuthSigs and bindingSig for you, by passing the reader as `context`. You'd parse the bundle's effecting data plus proof, construct a `Bundle<WithProofWithoutSignatures>` (or similar), then call `Bundle::try_authorize` with the remainder of `reader` to parse the signatures (relying on that API processing the actions in-order, followed by the bundle-level auth data, which matches the layout in the v5 transaction format).